packages/grafana-eslint-rules/tests/no-restricted-img-srcs.test.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable @grafana/no-restricted-img-srcs */
+import { RuleTester } from 'eslint';
+
+import noRestrictedImgSrcs from '../rules/no-restricted-img-srcs.cjs';
+
+RuleTester.setDefaultConfig({
+  languageOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+    parserOptions: {
+      ecmaFeatures: {
+        jsx: true,
+      },
+    },
+  },
+});
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('eslint no-restricted-img-srcs', noRestrictedImgSrcs, {
+  valid: [
+    {
+      name: 'uses build folder',
+      code: `const foo = 'public/build/img/checkbox.png';`,
+    },
+    {
+      name: 'uses import',
+      code: `
+import foo from 'img/checkbox.png';
+const bar = foo;
```

Comment:
Really minor, but this might be more useful if we changed it to `const bar = <img src={foo} />`